assets/haarcascade_conversor.js

```diff
@@ -0,0 +1,62 @@
+(function(){
+	'use strictc';
+
+	/**
+	* Convert json to be used by tracking.js
+	* @param {Object} json as file opencv_haarcascade_eye.js
+	* @returns {Array}
+	*/
+	var conversor = function(json){
```

Comment:
i think it may be better to have a more explicit name, and in english. It would be easier to understand for the user

Maybe something like haarcascade_converter.js and ```function convertHaarCascase(){}```